packages/dx-react-chart-material-ui/src/templates/series/slice.jsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import { withStyles } from 'material-ui/styles';
+import classNames from 'classnames';
+
+const styles = () => ({
+  root: {
+    stroke: 'none',
+  },
+});
+
+class SliceBase extends React.PureComponent {
+  render() {
+    const {
+      x, y, classes, className, ...restProps
+    } = this.props;
+    return (
+      <path
+        transform={`translate(${x} ${y})`}
+        className={classNames(classes.root, className)}
+        {...restProps}
+      />
+    );
+  }
+}
+
+SliceBase.propTypes = {
+  x: PropTypes.number.isRequired,
+  y: PropTypes.number.isRequired,
+  d: PropTypes.string.isRequired,
```

Comment:
Is the `d` prop useless for this component?

Reply:
No. It is the main property for a path.

Reply:
Seems in this scenario the `d` will spread with other rest props

Reply:
Yes, it is true. But this option is necessary.